dynamiqs/smesolve/sme_solver.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from math import sqrt
+
+import torch
+from torch import Tensor
+
+from ..mesolve.me_solver import MESolver
+from ..options import Options
+from ..utils.solver_utils import cache
+from ..utils.td_tensor import TDTensor
+from ..utils.utils import trace
+
+
+class SMESolver(MESolver):
+    def __init__(
+        self,
+        H: TDTensor,
+        y0: Tensor,
+        exp_ops: Tensor,
+        options: Options,
+        *,
+        jump_ops: Tensor,
+        meas_ops: Tensor,
+        etas: Tensor,
+        generator: torch.Generator,
+        t_save_y: Tensor,
+        t_save_meas: Tensor,
```

Comment:
I don't really understand how you chose keyword vs non-keyword arguments here. Since its an internal function, I maybe would not bother with the `*` and use only keyword arguments when defining the class instance

Reply:
The non-keyword arguments are the one of the base class.